proto/warden/sched/v1beta1/params.proto

```
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package warden.sched.v1beta1;
+
+import "amino/amino.proto";
```

Buf (1.47.2)

import "amino/amino.proto": file does not exist

(COMPILE)